ts/@live-compositor/core/src/live/output.ts

```diff
@@ -87,25 +90,18 @@ class Output {
     };
   }
 
-  public close(): void {
-    this.throttledUpdate = () => {};
-    // close will switch a scene to just a <View />, so we need replace `throttledUpdate`
-    // callback before it is called
-    this.outputShutdownStateStore.close();
+  public async close(): Promise<void> {
+    this.throttledUpdate.setFn(async () => {});
+    this.renderer.stop();
+    await this.throttledUpdate.waitForPendingCalls();
```

Comment:
I changed the throttledUpdate to class because I need to be able to wait for pending requests to finish, otherwise `LiveCompositor.terminate()` might kill the server before pending request are done